upgrade/upgrade-instructions-for-entities-simplification.md

```diff
@@ -153,6 +153,7 @@ and fix them appropriately (you can copy paste them from [Github](https://github
 - $productPriceCalculation = $this->getContainer()->get(ProductPriceCalculationForUser::class);
 - /** @var \Shopsys\FrameworkBundle\Model\Cart\Item\CartItemFactory $cartItemFactory */
 - $cartItemFactory = $this->getContainer()->get(CartItemFactory::class);
+- $cart = $cartFacade->getCartOfCurrentCustomerCreateIfNotExists();
```

Comment:
To be more clear we should leave the diff exactly same to the actual change. What do you think?

```suggestion
- $cart = $cartFacade->getCartOfCurrentCustomerCreateIfNotExists();
  $product = $productRepository->getById(1);
```